### ISSUE TITLE
Not getting good results

### ISSUE THREAD

**Issue by vp1993** [arahusky/Tensorflow-Segmentation#25]
Hi @arahusky, 

I trained the model using around 5,000 images, but somehow the loss is always around 0.42 and the results are not that impressive. 
My batch size is 100 and I train for 500 epochs.

Could you please tell me what could the problem be?
